src/main/java/org/ngafid/SendEmail.java

```diff
@@ -177,16 +211,30 @@ public static void sendEmail(ArrayList<String> toRecipients, ArrayList<String> b
 
                 // Set To: header field of the header.
                 for (String toRecipient : toRecipients) {
+
                     //list of users who do not want emails: TODO: make this a user setting
-                    if (toRecipient.equals("nievesn2@erau.edu")) continue;
+                    if (toRecipient.equals("nievesn2@erau.edu"))
```

Comment:
Once we land this, we can manually disable these settings for this user (as a courtesy) and then get rid of this.